pythonforandroid/recipe.py

```diff
@@ -375,6 +380,9 @@ def get_recipe_dir(self):
     # Public Recipe API to be subclassed if needed
 
     def download_if_necessary(self):
+        if self.ctx.ndk_api < self.min_ndk_api_support:
+            error(f"In order to build '{self.name}', you must set minimum ndk api (minapi) to `{self.min_ndk_api_support}`.\n")
+            exit(1)
```

Comment:
Wait, shouldn't we leverage `pythonforandroid/recommendations.py`?
Maybe @misl6 has better insights